build/simp-environment.spec

```diff
@@ -33,26 +33,24 @@
 
 Summary: The SIMP Environment Scaffold
 Name: simp-environment
-Version: 6.2.10
+Version: 6.3.0
 Release: 0%{?dist}
 License: Apache License 2.0
 Group: Applications/System
 Source: %{name}-%{version}-%{release}.tar.gz
 Buildroot: %{_tmppath}/%{name}-%{version}-%{release}-buildroot
 Requires: libselinux-utils
+Requires: openssl
 Requires: policycoreutils
-Requires: pupmod-simp-simp >= 3.0.0
-Requires: pupmod-simp-pki >= 6.0.0-0
-Requires: createrepo
+Requires: rubygem(simp-cli) >= 1.0.0-0
```

Comment:
Did you mean to require simp-adapter?  I don't understand why simp-cli is a dependency.